dashboards-observability/public/components/visualizations/charts/metrics/metrics.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+import React from 'react';
+import { sum, min, max, meanBy } from 'lodash';
+import { EmptyPlaceholder } from '../../../event_analytics/explorer/visualizations/shared_components/empty_placeholder';
+import './metrics.scss';
+
+export const Metrics = ({ visualizations }: any) => {
+  const {
+    data,
+    metadata: { fields },
+  } = visualizations.data.rawVizData;
+  const { dataConfig = {}, layoutConfig = {} } = visualizations.data.userConfigs;
+  const dataConfigTab =
+    visualizations.data?.rawVizData?.metrics?.dataConfig &&
+    visualizations.data.rawVizData.metrics.dataConfig;
+
+  const dataTitle = dataConfig?.panelOptions?.title;
+
+  const metricLabel = dataConfigTab?.metrics?.filter((value) => value.label !== '');
+
+  const getSelectedColorTheme = (field: any, index: number) =>
+    (dataConfig?.colorTheme?.length > 0 &&
+      dataConfig.colorTheme.find((colorSelected) => colorSelected.name.name === field)?.color) ||
+    '#000';
+  const fontSize = dataConfig?.fontSize?.fontSize ? dataConfig?.fontSize?.fontSize : 48;
```

Comment:
`dataConfig?.fontSize?.fontSize` question marks not required in the `if` block. If the condition is satisfied, the `fontsize` exists already.